docs/developer-docs/defi/overview.mdx

```diff
@@ -4,54 +4,137 @@ keywords: [intermediate, defi, overview, tokens, icrc-1, icrc ledger, icrc index
-way around. This provides the benefit of providing a certain degree of anonymity to the ICP ledger, but it also means that the ICP ledger can never have the same internal representation as ICRC-1 ledgers.
+### Difference between the ICP ledger and ICRC-1 ledgers
+
+The ICP ledger is a specific implement that follows no official standard. It was created before the ICRC-1 standard had been conceptualized and created.
+
+ICRC-1 is a standard, not a specific implementation of a ledger. There is a [reference implementation](https://github.com/dfinity/ic/tree/master/rs/rosetta-api/icrc1/ledger) created by DFINITY, but there may be different ICRC-1 ledgers with different implementations that all follow the same standard.
+
+The ICRC-1 reference implementation and the ICP ledger have different endpoints, different transaction and block objects, and quite a few other, more subtle, differences.
+
+The ICP ledger implements all endpoints defined in the ICRC-1 standard; however, this does not mean that the ICP ledger is an ICRC-1 ledger. The ICRC-1 standard clearly defines how an `Account` should be represented, while the ICP ledger uses `AccountIdentifiers`.
 
-In an attempt to comply with the ICRC-1 standard, the ICP ledger implements all endpoints defined in the ICRC-1 standard. However, this does not mean that the ICP ledger is an ICRC-1 ledger. The ICRC-1 standard clearly defines how accounts should be represented.
-The ICP ledger, due to its use of `AccountIdentifiers`, cannot fulfill this requirement. The ICP ledger successfully implements all ICRC-1 endpoints and they can be used for any other ICRC-1 endpoint.
 Should a future ICRC-1 extension standard or the use of index canisters result in the exposure of the internal representation of accounts, any ICRC-1 ledger will deviate from the ICP ledger.
 
-## Index canisters
-The ICP ledger lets you query created blocks through various endpoints. See the documentation on [how to interact with the ICP ledger for more info](/docs/current/developer-docs/defi/icp-tokens/using-the-ledger). If your ICRC-1 ledger supports the extension standard ICRC-3, then your ICRC-1 ledger will have an endpoint to serve transactions.
+See the documentation on [how to interact with the ICRC ledger for more info](/docs/current/developer-docs/defi/icrc-1/icrc-1-ledger-setup).
+
+## Indexes
+
+There are two different types of index canisters: one for the [ICP ledger](/docs/current/developer-docs/defi/tokens/indexes) and one for [ICRC-1 compatible ledgers](/docs/current/developer-docs/defi/tokens/indexes). Due to the previously mentioned difference between the ICP ledger and ICRC-1 ledgers, these two index canisters do not use identical endpoints. Since one uses `Account` and the other uses `AccountIdentifier` as an internal representation of accounts, the returned transactions will differ in this regard.
```

Comment:
It is unclear what "index" means and why the reader should care about it.

Reply:
Revised